src/building/how-to-build-and-run.md

```diff
@@ -26,13 +26,10 @@ cp config.toml.example config.toml
 ```
 
 Then you will want to open up the file and change the following
-settings (and possibly others, such as `llvm.ccache`):
+settings (and possibly others, such as `llvm.ccache`). Also, see the notes
+below about LLVM-related settings.
 
 ```toml
-[llvm]
-# Indicates whether the LLVM assertions are enabled or not
-assertions = true
```

Comment:
I remember talking with @nikomatsakis about this and him suggesting to leave this in. I'm also not sure if removing this is a better suggestion.

Reply:
I personally find this useful. It does make builds and bootstraps take longer, though, and it's often not relevant -- but when you're debugging changes to codegen or other LLVM-related problems, it's invaluable (and people often forget to enable it at that time).

Reply:
Sorry, looking over this PR again after a while. We add the suggestion to turn this on below. Are you suggesting that we suggest always turning it on (and if they use system LLVM it doesn't matter)?